Lib/posixpath.py

```diff
@@ -169,6 +169,15 @@ def islink(path):
         return False
     return stat.S_ISLNK(st.st_mode)
 
+
+# Is a path a junction?
+
+def isjunction(path):
+    """Test whether a path is a junction
+    Junctions are not a part of posix semantics"""
+    return False
```

Comment:
Calling `os.lstat()` implicitly calls `os.fspath()`, which raises `TypeError` if the path argument isn't a string, bytes, or path-like. Even though it isn't needed normally, `os.fspath()` still has to be called on POSIX.
   
```suggestion
    os.fspath(path)
    return False
```